tox.ini

```diff
@@ -100,7 +100,7 @@ commands =
 
     ; Run the tests wrapped using coverage.
     {withcov,coverage}: python {toxinidir}/admin/_copy.py {toxinidir}/admin/zz_coverage.pth {envsitepackagesdir}/zz_coverage.pth
-    {withcov,coverage}: coverage erase
+;    {withcov,coverage}: coverage erase
```

Comment:
Please revert this and let's discuss whether it's appropriate on a separate ticket :).